syncmap_test.go

```diff
@@ -0,0 +1,132 @@
+package dapper_test
+
+import (
+	"sync"
+	"testing"
+)
+
+type syncmaps struct {
+	Map sync.Map
+}
+
+// This test verifies that that sync.Map key/value types are rendered when
+// they can be inferred from the context.
+func TestPrinter_SyncMap(t *testing.T) {
+	var m sync.Map
+
+	test(t, "empty sync.SyncMap", &m, "*sync.Map{}")
+
+	m.Store(1, 100)
+	m.Store(2, 200)
+
+	test(
+		t,
+		"*sync.SyncMap",
```

Comment:
```suggestion
		"sync.Map",
```

This is just the test description -- even though we're testing using a pointer, the test is still just for `sync.Map`.